lib/main.js

```diff
@@ -61,7 +60,9 @@ export function consumeStatusBar (statusBar) {
     statusBarItem.setLineEndings(lineEndings)
   }
 
-  let debouncedUpdateTile = _.debounce(updateTile, 0)
```

Comment:
Even with a timeout of `0`, debounce will prevent multiple calls to `debouncedUpdateTile` in the same call stack from ultimately called `updateTile` more than once. Switching to a pure `setTimeout(..., 0)` no longer debounces calls to this function done in the same call stack.

The use of `debouncedUpdateTile` below seems like it will work the same, but I'd remove the `debounce` name because it no longer is debounced. This is just deferred (also in Underscore as [`_.defer`](http://underscorejs.org/docs/underscore.html#section-81)).